environments/kolla/configuration.yml

```diff
@@ -76,6 +76,18 @@ horizon_backend_database: "yes"
 
 # keystone
 keystone_token_provider: "fernet"
+keystone_identity_providers:
+  - name: "gx-scs"
```

Comment:
Would prefer to use `keycloak` as a more generic name.

Reply:
Done